package-res/resources/web/pentaho/type/SpecificationScope.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright 2010 - 2015 Pentaho Corporation.  All rights reserved.
```

Comment:
Copyright date and the extra space. Your chance to try @davidmsantos90 's magic regex.